accounts/ethkey/src/brain_prefix.rs

```diff
@@ -39,15 +39,11 @@ impl BrainPrefix {
 	pub fn phrase(&self) -> &str {
 		&self.last_phrase
 	}
-}
-
-impl Generator for BrainPrefix {
-	type Error = Error;
 
-	fn generate(&mut self) -> Result<KeyPair, Error> {
+	pub fn generate(&mut self) -> Result<KeyPair, Error> {
```

Comment:
Moved this to a free function as it needs to return an error, whereas the `Generate` trait changed (it can't fail now).